src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -70,21 +70,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_PlatformWithoutConfigurationInference>$(Platform)</_PlatformWithoutConfigurationInference>
   </PropertyGroup>  
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'x64' ">
-    <PlatformTarget>x64</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">x64</PlatformTarget>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'x86' ">
-    <PlatformTarget>x86</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">x86</PlatformTarget>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'ARM' ">
-    <PlatformTarget>ARM</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">ARM</PlatformTarget>
   </PropertyGroup>
 
   <!-- Default settings for all projects built with this Sdk package -->
   <PropertyGroup>
-    <DebugType>portable</DebugType>
+    <DebugType Condition=" '$(DebugType)' == '' ">portable</DebugType>
 
     <!-- This will turn off the base UWP-specific 'ResolveNuGetPackages' target -->
-    <ResolveNuGetPackages>false</ResolveNuGetPackages>
+    <ResolveNuGetPackages >false</ResolveNuGetPackages>
```

Comment:
I take it you changed your mind on letting this be set by user leading to this whitespace? Should it go in targets (after user code)? @natidea is this still what triggers the targets from not getting imported at all with your subsequent changes?